config.default

```diff
@@ -326,6 +326,15 @@ movingAveragePoints = 12
 # regionIndicesToPlot should be an integer
 regionIndicesToPlot = 5
 
+# Data source to read for comparison.  There are three options
+# 1 - ERS SSTv3b -- Common in CVDP analysis -- 1854 - 2016
+# 2 - ERS SSTv4 -- Updated version of previous -- 1854 - 2016
+# 3 - HADIsst -- Hadley center analysis -- 1870 - 2016
+# Valid Choices -- Uncomment your choice 
+#observationData = /global/homes/l/lvroekel/HADIsst_nino34.nc
+observationData = /global/homes/l/lvroekel/ERS_SSTv3b_nino34.nc
+#observationData = /global/homes/l/lvroekel/ERS_SSTv4_nino34.nc
```

Comment:
@xylar I agree with you that specifying the name is better, I will fix this.  Regarding the last point, I'm not sure how the obs flag would work with multiple panels.  I have a hard time seeing how to plot or not plot OBS with specifying subplot panels.  I'm sure I'm missing a simple solution though.